frappe/utils/jinja.py

```diff
@@ -211,7 +208,14 @@ def set_filters(jenv):
 	if frappe.flags.in_setup_help: return
 
 	# load jenv_filters from hooks.py
+	for filter_name, filter_function in get_jenv_customization("filters"):
+		jenv.filters[filter_name] = frappe.get_attr(filter_function)
+
+def get_jenv_customization(customizable_type):
+	import frappe
 	for app in frappe.get_installed_apps():
```

Comment:
site may not be always initiated when this is called. So important to check `if getattr(frappe.local, "site", None):`